src/App.tsx

```diff
@@ -172,6 +172,15 @@ function AppContent() {
   const [plotNames, setPlotNames] = useState<Record<number, string>>({});
   const [showNavBar, setShowNavBar] = useState(true);
 
+  const setDefaults = useCallback(() => {
+    if (!isPlaceholderOrg(selectedOrganization.id)) {
+      const savedColumns = orgPreferences.accessionsColumns ? (orgPreferences.accessionsColumns as string[]) : [];
+      const defaultColumns = savedColumns.length ? savedColumns : DefaultColumns(preferredWeightSystem).fields;
+      setAccessionsDisplayColumns(defaultColumns);
```

Comment:
> Looks good in Vercel

Thanks @charlie-tf

Reply:
> Does this live in recoil?

The columns, search criteria etc. are in the App.tsx state.. and passed down to the accessions table.
Every time we update them, the entire app renders (not just the accessions table).
I'd like to move this state into the accessions table and not expose it outside. 
Will do that as a separate effort. cc @constanzauanini is it ok to move it into accessions table? The current code has been like that since before my time, I think maybe it's ok to change it now?

Reply:
Ugh, ok nvm.. i think it's because we want to keep this state within the sesssion.. example if user navigates to another menu and returns to accessions.. the last used filters stick (within the session, not persisted across sessions).

Reply:
Yes, the reason is that, but if we are saving the columns in preferences we could probably remove it from App.tsx ?

Reply:
good point.. we do need the filters (search criteria) in the App (so it's preserved across navigation).. i will think about moving the columns alone into the accessions table maybe. was hoping to move them all but that will break.